src/pocketmine/lang/locale/fra.ini

```diff
@@ -77,31 +77,31 @@ commands.generic.exception=Une erreur inconnue est survenue lors de la tentative
 commands.generic.permission=Vous n'avez pas la permission d'utiliser cette commande
 commands.generic.notFound=Cette commande n'est pas reconnue : faites /help pour obtenir la liste des commandes
 commands.generic.player.notFound=Le joueur n'a pas été trouvé
-commands.generic.usage=Usage: {%0}
-commands.generic.level=nom-niveau
-commands.generic.seed=nom-seed
-commands.generic.name=nom
-commands.generic.generator=nom-générateur
+commands.generic.usage=Utilisation : {%0}
+commands.generic.level=Monde
+commands.generic.seed=Semencd
```

Comment:
PLEASE change "Semance" par "Seed ou Graine" But Not "Semance" ! (Cc @FaanMaario)